plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/CubridConstants.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid;
+
+import org.jkiss.dbeaver.ext.generic.GenericConstants;
+
+public class CubridConstants extends GenericConstants {
+	
+	public static final String OWNER_NAME = "owner_name";
+	public static final String CLASS_NAME = "class_name";
+	public static final String IS_SYSTEM_CLASS = "is_system_class";

```

Comment:
I think it's better not to extend GenericConstants but to use them directly when required.

Reply:
I have fixed it.